src/openms/source/CHEMISTRY/ElementDB.cpp

```diff
@@ -574,17 +591,34 @@ namespace OpenMS
 
   void ElementDB::addElementToMaps_(const string& name, const string& symbol, const unsigned int an, const Element* e)
   {
-    names_[name] = e;
-    symbols_[symbol] = e;
-    atomic_numbers_[an] = e;
+    #pragma omp critical(OpenMS_ElementDB)
```

Comment:
why this critical region?
this is the only place that reading/writing is protected, so it does not really help a lot.
Adding elements should not be done in a threaded context anyway IMHO.

Reply:
@cbielow I agree, it does not prevent concurrent reading but it does help against multiple people writing at the same time which should be helpful